app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t(:base_title)
```

Comment:
trong helper và model thì khai báo cụ thể nhé
```suggestion
    base_title = I18n.t(:base_title)
```